app/services/api/v3/ecf/participants_query.rb

```diff
@@ -18,7 +19,8 @@ def participants_for_pagination
                     .joins(participant_profiles: :induction_records)
                     .joins("JOIN (#{latest_induction_records_join.to_sql}) AS latest_induction_records_join ON latest_induction_records_join.latest_id = induction_records.id")
                     .distinct
-          scope = updated_since_filter.present? ? scope.where(users: { updated_at: updated_since.. }) : scope
+          scope = scope.where(users: { updated_at: updated_since.. }) if updated_since_filter.present?
+          scope = scope.where(participant_profiles_users: { training_status: }) if training_status.present?
```

Comment:
Could we use the latest induction record training status not the participant profile training status as it needs to be scoped to provider unfortunately?